app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
```

Comment:
- [ ] Like the second reviewer I was not able to sign up a user to test the first reviewer's comment. Kindly ensure you fix the error. There is an error regarding the `posts_counter` that is not being handled when a user is being created.

  I fixed this error by changing line 9 in this User model from this:
  ```
  attribute :post_counter, :integer
  ```
  to something like this:
  ```
  attribute :post_counter, default: -> { 0 }
  ```
  This sets the `post_counter` to default of 0, before getting to the validation in line 12. This will fix the error. And yes, the email is confirmed before logging in, good job.

  ![image](https://github.com/ahmidou09/blog_app/assets/54848363/e13c0970-2079-420d-9a2a-d84dae189d84)